app/views/locations/new.html.erb

```diff
@@ -20,7 +20,7 @@
                              width: 5 %>
 
       <%= f.govuk_text_field :name,
-                             label: { text: "Location name (optional)", size: "m" },
+                             label: { text: "Name (optional)", size: "m" },
```

Comment:
Why is this reverting? Didn't we just change this?

Reply:
Its per prototype, can you confirm this please @paulrobertlloyd